LongWarOfTheChosen/Config/XComAI.ini

```diff
@@ -155,6 +174,92 @@
+; ----------------------------------------------------------------------------------------------------------
++NewBehaviors=(BehaviorName=PriestRedLastActionSelector, NodeType=Selector, \\
+Child[0]=TryHighPriorityShot, \\
+Child[1]=TryStasis, \\
+Child[2]=TryPsiMindControl, \\
+Child[3]=MoveDefensiveWithLoS, \\
+Child[4]=HuntEnemyWithCover)
++NewBehaviors=(BehaviorName=PriestRedFirstActionSelector, NodeType=Selector, \\
+Child[0]=DoIfFlankedMove, \\
+Child[1]=TryStasis, \\
+Child[2]=TryHighPriorityShot, \\
+Child[3]=TryPsiMindControl, \\
+Child[4]=TryHolyWarrior, \\
+Child[5]=TryShootOrReloadOrOverwatch, \\
+Child[6]=HuntEnemyWithCover)
+
++NewBehaviors=(BehaviorName=PriestRedFirstAction, NodeType=Sequence, Child[0]=NotLastActionPoint, Child[1]=PriestRedFirstActionSelector)
++NewBehaviors=(BehaviorName=PriestRedLastAction, NodeType=Sequence, Child[0]=IsLastActionPoint, Child[1]=PriestRedLastActionSelector)
+
+
++BehaviorRemovals="SelectPriestAbility"
++NewBehaviors=(BehaviorName=SelectPriestAbility, NodeType=Selector, Child[0]=MimicBeaconBehavior, Child[1]=PriestRedFirstAction, Child[2]=PriestRedLastAction, Child[3]=AdventRedFallbackAction)
+
++NewBehaviors=(BehaviorName=TargetAffectedByEffect-TransferMecToOutpost, NodeType=Condition)
++NewBehaviors=(BehaviorName=TargetScoreByScaledDistance, NodeType=Action, Param[0]="1")
++NewBehaviors=(BehaviorName=ScoreTargetIfMindControlledPositive, NodeType=Sequence, Child[0]=TargetAffectedByEffect-MindControl, Child[1]=AddToTargetScore_500)
++NewBehaviors=(BehaviorName=ScoreTargetIfFullOverridden, NodeType=Sequence, Child[0]=TargetAffectedByEffect-TransferMecToOutpost, Child[1]=AddToTargetScore_-1000)
++NewBehaviors=(BehaviorName=AvoidKillableOrFlanked, NodeType=Sequence, Child[0]=TargetKillableOrFlanked, Child[1]=AddToTargetScore_-1000)
++NewBehaviors=(BehaviorName=PrioritizeNotTakeCover, NodeType=Sequence, Child[0]=TargetDoesNotTakeCover, Child[1]=AddToTargetScore_30)
+
++NewBehaviors=(BehaviorName=SSScoreTargetIfMindControlledPositive, NodeType=Successor, Child[0]=ScoreTargetIfMindControlledPositive)
++NewBehaviors=(BehaviorName=SSScoreTargetIfFullOverridden, NodeType=Successor, Child[0]=ScoreTargetIfFullOverridden)
++NewBehaviors=(BehaviorName=SSAvoidKillableOrFlanked, NodeType=Successor, Child[0]=AvoidKillableOrFlanked)
++NewBehaviors=(BehaviorName=SSPrioritizeNotTakeCover, NodeType=Successor, Child[0]=PrioritizeNotTakeCover)
+
+
++BehaviorRemovals="EvaluateForStasis"
++NewBehaviors=(BehaviorName=EvaluateForStasis, NodeType=Sequence, Child[0]=TargetIsNotCivilian, Child[1]=TargetNotTeamResistance, Child[2]=TargetIsEnemy, Child[3]=TargetIsNotEnemyVIP, Child[4]=TargetNotBoundOrPanicked, Child[5]=TargetIsAttackable, Child[6]=TargetScoreByScaledDistance, Child[7]=AddToTargetScore_-7, Child[8]=SSScoreTargetIfMindControlledPositive, Child[9]=SSScoreTargetIfFullOverridden, Child[10]=SSAvoidKillableOrFlanked, Child[11]=SSPrioritizeNotTakeCover)
```

Comment:
Do these checks take into account which team the target is on? Or I guess the Priest's Stasis can't target units on their own team.

Reply:
Yes, stasis can't be used on allies without stasis shield